src/main/java/org/scenarioo/api/util/xml/ScenarioDocuXMLFileUtil.java

```diff
@@ -99,5 +114,23 @@
 		}
 		return result;
 	}
-	
+
+	/**
+	 * Read all passed files and unmarshall these to the given object type, also return the name of the directory the
+	 * object was loaded from.
+	 */
+	public static <T> List<ObjectFromDirectory<T>> unmarshalListOfFilesWithDirNamesIgnoreInvalid(
+		final List<File> files, final Class<T> targetClass, final Class<?>... classesToBind) {
+		List<ObjectFromDirectory<T>> result = new ArrayList<ObjectFromDirectory<T>>();
+		for (File file : files) {
+			try {
+				result.add(new ObjectFromDirectory<T>(unmarshal(targetClass, file), file
+					.getParentFile().getName()));
+			} catch (Throwable e) {
+				LOGGER.error("Could not parse invalid File: " + file.getAbsolutePath() + " - will ignore this object.");
```

Comment:
warn instead of error?

Reply:
see above, I think there is something wrong with the system but we only log it, because we do not want to not run at all int hat case, so error seems ok to me. Somebody should really take care of this problem.